sdcm/nemesis.py

```diff
@@ -1831,12 +1831,21 @@ def repair_nodetool_rebuild(self):
 
     def disrupt_nodetool_cleanup(self):
         # This fix important when just user profile is run in the test and "keyspace1" doesn't exist.
-        test_keyspaces = self.cluster.get_test_keyspaces()
-        for node in self.cluster.nodes:
+        # test_keyspaces = self.cluster.get_test_keyspaces()
+
+        # Inner disrupt function for ParallelObject
+        def _dis_nodetool_cleanup(node):
             InfoEvent('NodetoolCleanupMonkey %s' % node).publish()
             with adaptive_timeout(Operations.CLEANUP, node, timeout=HOUR_IN_SEC * 48):
-                for keyspace in test_keyspaces:
-                    node.run_nodetool(sub_cmd="cleanup", args=keyspace)
+                # for keyspace in test_keyspaces:
+                node.run_nodetool(sub_cmd="cleanup")
+
+        threads = ParallelObject(self.cluster.nodes, num_workers=min(32, len(self.cluster.nodes)))
+        thread_results = threads.run(_dis_nodetool_cleanup, ignore_exceptions=True, unpack_objects=True)
+        if any(res for res in thread_results if res.exc):
+            self.log.error("Nodetool cleanup has failed", exc_info=ParallelObjectException(results=thread_results))
```

Comment:
no point logging error like this, no one gonna notice it, error should always strive to raise exceptions

Reply:
I remove this code. If an exception occurs it will be raised by parallelObject.run() inner logic:

```
runs_that_finished_with_exception = [res for res in results if res.exc]
        if runs_that_finished_with_exception:
            raise ParallelObjectException(results=results)
```